stdlib/http/src/main/ballerina/src/http/cookie/cookie.bal

```diff
@@ -66,7 +66,7 @@ public type Cookie object {
 
     # Checks the validity of the attributes of the cookie.
     #
-    # + return  - set to `true` if the attributes of the cookie are in the correct format. Otherwise, returns an `error`.
+    # + return  - `true` if the attributes of the cookie are in the correct format. Otherwise, returns an `error`.
```

Comment:
```suggestion
    # + return  - `true` if the attributes of the cookie are in the correct format. Otherwise, returns an `error`.
```
Same as above.